examples/src/components/AttributeElementsExample.jsx

```diff
@@ -19,74 +19,66 @@ export class AttributeFilterItem extends Component {
-                            isLoading: {isLoading.toString()}<br />
-                            offset: {offset}<br />
-                            count: {count}<br />
-                            total: {total}<br />
-                            nextOffset: {offset + count}
-                        </pre>
-                        <div>
-                            {validElements ? validElements.items.map(item => (
-                                <AttributeFilterItem
-                                    key={item.element.uri}
-                                    uri={item.element.uri}
-                                    title={item.element.title}
-                                />
-                            )) : null}
-                        </div>
-                        {validElements ? <pre>{JSON.stringify(validElements, null, '  ')}</pre> : null}
-                    </div>
-                );
-            }
-        };
+    getInnerComponent({ validElements, loadMore, isLoading, error }) {
+        const {
+            offset = null,
+            count = null,
+            total = null
+        } = validElements ? validElements.paging : {};
+        if (error) {
+            return <div>{error}</div>;
+        }
+        return (
+            <div>
+                <p>
+                    Use children function to map {'{'} validElements, loadMore, isLoading {'} '}
+                    to your React components.
+                </p>
+                <button
+                    className="button button-secondary s-show-more-filters-button"
+                    onClick={loadMore}
+                    disabled={isLoading || (offset + count === total)}
+                >More</button>
+                <h2>validElements</h2>
+                <pre>
+                    isLoading: {isLoading.toString()}<br />
+                    offset: {offset}<br />
+                    count: {count}<br />
+                    total: {total}<br />
+                    nextOffset: {offset + count}
+                </pre>
+                <div>
+                    {validElements ? validElements.items.map(item => (
+                        <AttributeFilterItem
+                            key={item.element.uri}
+                            uri={item.element.uri}
+                            title={item.element.title}
+                        />
+                    )) : null}
+                </div>
+                {validElements ? <pre>{JSON.stringify(validElements, null, '  ')}</pre> : null}
+            </div>
+        );
+    }
 
-        return (<div style={{ minHeight: 500 }}>
-            <AttributeElements
-                {...attributeElementsProps}
-            />
-        </div>);
+    render() {
+        return (
+            <div style={{ minHeight: 500 }}>
+                <AttributeElements identifier={employeeNameIdentifier} projectId={projectId} options={{ limit: 20 }}>
+                    {this.getInnerComponent}
```

Comment:
I would expect this to be `this.getInnerComponent()`

Reply:
AttributeElements uses a children function, not an element, so this.getInnerComponent is correct, imho.